### DISCUSSION TITLE
Turn on multigrid and frameless

### DISCUSSION THREAD

**Discussion by anson-ryea** [neovide/neovide#1059] [Q&A] [answered]
Is it possible to set multigrid and frameless to enabled in the init.vim? If yes, how to do it?

**Comment by Kethku** [comment, marked as answer]
Multigrid is a ui extension level feature, so its impossible to set dynamically. It must be specified either via a command line switch or an environment variable.

Frameless is similar in that its set at windows construction time. However its possible with some effort it could be changed to allow recreating the window on demand. But thatd be some work to get right.
